ha/core/event_manager/event_manager.py

```diff
@@ -332,4 +345,16 @@ def message_type(self, component: str) -> str:
         Args:
             component (str): component name.
         """
+        key = const.EVENT_MGR_MESSAGE_TYPE_KEY.replace("<component_id>", component)
```

Comment:
```
def publish(self, component:str, event: RecoveryActionEvent) -> None:
```
return type not matching
i think publish job is to send event in message bus

Reply:
this is message_type function , not publish